federation/pkg/federation-controller/service/servicecontroller.go

```diff
@@ -790,17 +699,6 @@ func (s *ServiceController) updateFederatedService(fedService *v1.Service, newLB
 		}
 	}
 
-	// Ensure DNS records based on Annotations in federated service for all federated clusters
-	if needUpdate && wantsDNSRecords(fedService) {
```

Comment:
@quinton-hoole, We may not need this anymore. We look for service status and update the federated ingress annotation. In case of non-loadbalancer type of service, service status will have empty LoadBalancer field and so we don't update the ingress annotation. is this ok?